defaults/test_default.yaml

```diff
@@ -5,7 +5,7 @@ test_duration: 60
 ip_ssh_connections: 'private'
 
 scylla_repo: ''
-scylla_repo_m: 'http://downloads.scylladb.com.s3.amazonaws.com/rpm/centos/scylla-2020.1.repo'
+scylla_repo_m: 'http://downloads.scylladb.com.s3.amazonaws.com/rpm/centos/scylla-2021.1.repo'
```

Comment:
following @roydahan 's suggestion, we could leave this one as it was, and only have the 2021 for the ones in the specific files

Reply:
Great suggestion!

Reply:
Bad suggestion!
It’s very confusing and not serving the purpose.

Reply:
The idea is that “new” OS version assume that you use newest release.

Reply:
this is exactly what is happening in here, and the other ones (older OSes) will use still 2020.1